src/main/java/com/savvato/tribeapp/controllers/AttributesAPIController.java

```diff
@@ -52,6 +54,19 @@ public ResponseEntity<List<AttributeDTO>> getAttributesForUser(
     else return ResponseEntity.status(HttpStatus.BAD_REQUEST).body(null);
   }
 
+  @GetUserPhrasesToBeReviewed
+  @GetMapping("/in-review/{userId}")
+  public ResponseEntity<List<ToBeReviewedDTO>> getUserPhrasesToBeReviewed(
+          @Parameter(description = "User ID of user", example = "1")
+          @PathVariable Long userId) {
+    List<ToBeReviewedDTO> rtn =
+            reviewSubmittingUserService.getUserPhrasesToBeReviewed(userId);
+    if(rtn.isEmpty()) {
```

Comment:
I believe this will return a BAD_REQUEST if there are no phrases to be reviewed? We want to return an OK with an empty list instead.

Reply:
updated to always return list.